doc/guides/writing-tests.md

```diff
@@ -2,9 +2,9 @@
 
 ## What is a test?
 
-A test must be a node script that exercises a specific functionality provided
-by node and checks that it behaves as expected. It should exit with code `0` on success,
-otherwise it will fail. A test will fail if:
+Most tests in Node.js core are JavaScript programs that exercise a specific
```

Comment:
Nit: Can you say `a functionality`? I feel like it should be `a piece of functionality` or just `functionality`.